primitives/src/ceremonies.rs

```diff
@@ -347,6 +348,22 @@ pub struct ReputationCacheValue {
 	pub reputation: Vec<(CeremonyIndexType, CommunityReputation)>,
 }
 
+#[derive(Encode, Decode, Copy, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo, MaxEncodedLen)]
+#[cfg_attr(feature = "serde_derive", derive(Serialize, Deserialize))]
+pub enum MeetupResult {
+	Ok,
+	VotesNotDependable,
+	MeetupValidationIndexOutOfBounds,
+	NoResult,
+	Other,
+}
+
+impl Default for MeetupResult {
+	fn default() -> Self {
+		MeetupResult::NoResult
+	}
+}
```

Comment:
If we remove the two unused variants above, I think we remove the default implementation.